Libraries/LibCrypto/SecureRandom.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2024, the Ladybird developers.
```

Comment:
to double check, you're sure you don't want your own copyright here?

Reply:
Yep, all good